Cli automatic answers to questions (#668)
openSUSE/agama: doc/cli_guidelines.md

## Non Interactive Mode

> There are questions subcommand that support `mode` subcommand to set non-interactive mode. Another option is to use `answers` subcommand to pass file with

Reviewer: ```suggestion
There is a `questions` subcommand that implements a `mode` subcommand to set the non-interactive mode. Another option is to use `answers` subcommand to pass a file with
```

> There are questions subcommand that support `mode` subcommand to set non-interactive mode. Another option is to use `answers` subcommand to pass file with

Reviewer 2: NP: I woud rephase it as: There is a `questions` subcommand that support `mode` and `answers` subcommands. The `mode` subcommand allows to set `interactive` and `no-interactive` modes. And `answers` allows to pass a file with prefefined answers.
